clang/test/CodeGenSYCL/debug-info-srcpos-kernel.cpp

```diff
@@ -9,7 +9,7 @@
 // 'kernel' template function on line 15 in this file).
 //
 
-#include "Inputs/sycl.hpp"
+#include "Inputs/sampler.hpp"
```

Comment:
Why this change?

Reply:
I figured out the root cause for this test but I don't know how to fix it. The issue is that I have added a default constructor to the stream class in the file `Inputs/sycl.hpp` and that breaks this test for some reason unknown to me. Ideally, this should not happen as the purpose of this test seems to be unrelated to the nature of my changes but here we are. So I created a special definition for sampler without a default constructor.

However, now that I think of it its probably best to just create an XFAIL and link an issue to it because this should definitely be looked at.

Reply:
Why can't it be looked at as a part of this PR? The PR should not break tests and XFAIL them because the reason why it started failing is not entirely known.

Reply:
Ok, with some help I was able to figure out what was going on. My changes were causing a reordering of the output that was being checked by the test so I had to alter the FileCheck CHECK commands slightly. The test passes now.